lib/events/index.js

```diff
@@ -602,163 +602,167 @@ export default function (self) {
           bottom: Math.max(self.dragStartObject.rowIndex, cell.rowIndex),
           right: Math.max(self.dragStartObject.columnIndex, cell.columnIndex),
         };
-        if (self.dragStartObject.columnIndex === -1) {
-          sBounds = self.getSelectionBounds();
-          dragBounds.left = -1;
-          dragBounds.right = s.length - 1;
-          dragBounds.top = Math.min(sBounds.top, cell.rowIndex);
-          dragBounds.bottom = Math.max(sBounds.bottom, cell.rowIndex);
-          if (dragBounds.top < 0) dragBounds.top = 0;
-        }
+        if (self.movingSelectionHandle) {
```

Comment:
I was talking about this line, which takes over the `mousemove` event when a selection handle is being dragged.

Reply:
I converted it to an event-based format, and moved scroll logic outside `self.selecting` behavior. Everything works fine now.